packages/events/src/EventSystem.ts

```diff
@@ -106,6 +173,25 @@ export class EventSystem implements ISystem<EventSystemOptions>
     /** The renderer managing this {@link PIXI.EventSystem}. */
     public renderer: IRenderer;
 
+    /**
+     * The event features that are enabled by the EventSystem
+     * This option only is available when using **@pixi/events** package
+     * (included in the **pixi.js** and **pixi.js-legacy** bundle), otherwise it will be ignored.
+     * @since 7.2.0
+     * @example
+     * const app = new PIXI.Application()
+     * app.renderer.events.features.globalMove = false
+     *
+     * // to override all features use Object.assign
+     * Object.assign(app.renderer.events.features, {
+     *  move: false,
+     *  globalMove: false,
+     *  click: false,
+     *  wheel: false,
+     * })
+     */
+    public features: EventSystemFeatures;
```

Comment:
Making this readonly would prevent user setting this prop (but would still allow members of EventSystemFeatures to be set).
```suggestion
    public readonly features: EventSystemFeatures;
```